packages/carbon-web-components/src/components/tooltip/tooltip.scss

```diff
@@ -18,13 +20,30 @@ $caret-horizontal-position: calc(#{$caret-size * (-1)} + 1px);
 @use '@carbon/styles/scss/utilities';
 @use '@carbon/styles/scss/spacing' as *;
 @use '@carbon/styles/scss/theme' as *;
-@use '@carbon/styles/scss/components/tooltip';
+@use '@carbon/styles/scss/components/tooltip/index' as *;
+@use '@carbon/styles/scss/components/popover/index' as *;
+@use '@carbon/styles/scss/utilities/button-reset';
 
 // https://github.com/carbon-design-system/carbon/issues/11408
 // @include tooltip;
 
 :host(#{$prefix}-tooltip) {
   @extend .#{$prefix}--tooltip__trigger;
+
+  .sb-tooltip-trigger {
```

Comment:
should this be a specific storybook style? that's what i'm seeing here: https://github.com/carbon-design-system/carbon/blob/db973afb60d77c84c39c102c874b3d98513957a0/packages/react/src/components/Tooltip/story.scss